### ISSUE TITLE
Prediction result fonts are too small

### ISSUE THREAD

**Issue by SamiurRahman1** [facebookresearch/SlowFast#393]
Hello, i am running demo using x3d models on custom videos. The prediction results on the top left corner are too small. is there any way i can enlarge the fonts?

Thanks

**Comment by nicholasguimaraes** [comment]
Yes, I am pretty sure it is in slowfast/visualization/video_visualizer.py

Function draw_text , line 64. 

`def draw_text(
        self,
        text,
        position,
        *,
        font_size=None,
        color="w",
        horizontal_alignment="center",
        vertical_alignment="bottom",
        box_facecolor="black",
        alpha=0.5,
    ):`
